microsoft/atlas-design
# Hero Component Variant

Task: task-953272

Link: preview-616

Adding a style variant to the Hero component to account for right aligned card-like content.   This was a result of the Skilling Plan efforts and designs.

## Testing
1. Pull down atlas repo and checkout this branch.
1. Run the branch.
1. Open the Hero Component and Patterns pages
1. Scroll down to the Hero components.
    - Expected result: 
      - Hero with right align content to appear.
1. Open the pages in full screen and swap the themes as you see fit. Also resize the screen to mobile and tablet widths
    -  Expected result:
      - The card and hero should reflect theme changes and remain visible at all times.

## Additional information

[Optional]

## Contributor checklist
